packages/main/src/plugin/color-registry.spec.ts

```diff
@@ -30,6 +30,8 @@ import type { ColorDefinition } from '/@api/color-info.js';
 import type { RawThemeContribution } from '/@api/theme-info.js';
 
 import colorPalette from '../../../../tailwind-color-palette.json';
+import { isWindows } from '../util.js';
+import * as util from '../util.js';
```

Comment:
I think there is a linting rule to only have one line when importing from the same location

Reply:
And I've also missed the test on the bottom

Reply:
@benoitf could you please check?